test/JDBC/input/babel_varbinary.sql

```diff
@@ -58,6 +58,28 @@ go
 select 10 / 0x00;
 go
 
+--division between varbinary and int4 datatype, vice-versa
+select @@microsoftversion / 0x1000000
```

Comment:
tests cases with binary value > INT4 binary value < -INT4. Similarly MIN/MAX for int argument, what if int value is greater than INT4/-INT4

Reply:
Addressed